src/App.js

```diff
@@ -1,39 +1,80 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
+import UpperBoard from "./UpperBoard";
+// import MyTimer from "./Timer.js";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+  const [lions, setLions] = useState(0);
+  const [tigers, setTigers] = useState(0);
+
+  const teamHandler = (set, value, inc) => {
+    return () => set(value + inc);
+  };
```

Comment:
Good work on this higher order function, I know it is really weird to get used to!